src/index.tsx

```diff
@@ -42,7 +43,9 @@ const Terminal = (props: Props) => {
 
   // We use a hidden input to capture terminal input; make sure the hidden input is focused when clicking anywhere on the document
   useEffect(() => {
-    document.onclick = () => document.getElementById("terminal-hidden")?.focus()
+    if(props.autoFocus) {
+        document.onclick = () => document.getElementById("terminal-hidden")?.focus()
+    }
```

Comment:
How is terminal focus regained after losing focus? When I was testing this change out, I clicked outside of the terminal, which lost focus, then was unable to type anything in the terminal because there was no way to regain focus.

Reply:
I wonder if a better solution to this problem would be to add a ref to the entire `div.react-terminal-wrapper` and instead of
```
 document.onclick = () => document.getElementById("terminal-hidden")?.focus()
    }
 ```
 do somethng like:
 ```
 reactTerminalRef.current.onClick = () => document.getElementById("terminal-hidden")?.focus()
    }
 ```
 This would allow focus to be regained when clicking the terminal area, but not when clicking outside of it.

Reply:
Hi @jonmbake sorry for the late response. I use the terminal UI only as output not as input, so never need to focus the terminal but your solution appear more general and better for the project, maybe can be a mix of two, focus only when clicked and the ability to never focus because never need the user to enter data.   

I also prefer the second solution, on document click can be problematic for the most of the project with more elements.

Reply:
Thanks for the response, @DonsWayo.

> use the terminal UI only as output not as input

Interesting. If you have the `onInput` prop set `== null`. Another fix for this use case would be:
```
    if(props.onInput != null) {
        document.onclick = () => document.getElementById("terminal-hidden")?.focus()
    }
```

I'll open up an issue to improve this focusing and _scroll into view_ behavior.